### DISCUSSION TITLE
Current Xcode version not available?

### DISCUSSION THREAD

**Discussion by dmelgar** [actions/virtual-environments#6021] [Installed software]
is there a support of policy statement on how quickly new/current versions of Xcode are available?

As of today, the current Xcode version is 13.4.1 yet the newest version on GitHub actions appears to be 13.2.1.
There have been format changes for spm resolve files and current apps won't build on 13.2.1.

Is there a way to predict how far behind GitHub actions lags?

Is there a way to install the current Xcode version for a build/runner?

Thanks

**Comment by mikhailkoliada** [comment]
@dmelgar Hi

Xcode 13.4.x is only supported on macOS-12 and we have it [installed](https://github.com/actions/virtual-environments/blob/main/images/macos/macos-12-Readme.md)

**Reply by dmelgar** [reply]
Ok, thanks!
Why is macos-latest macOS-11? MacOS 12 has been GA for almost a year.
I would never have guessed is needed to be explicitly stated.
